source/module_base/test/tool_threading_test.cpp

```diff
@@ -54,3 +56,47 @@ TEST(tool_threading, TASK_DIST_1D_test_longlong)
     EXPECT_EQ(st,6);
     EXPECT_EQ(le,6);
 }
+
+TEST(tool_threading, BLOCK_TASK_DIST_1D_test_int)

```

Comment:
Underscores are not suggested to be used in naming unit tests. Hump-shaped naming style is preferred.

Reply:
> Underscores are not suggested to be used in naming unit tests. Hump-shaped naming style is preferred.

OK. I'll change it.